nautobot_floor_plan/app-config-schema.json

```diff
@@ -11,7 +11,26 @@
         "default_y_axis_labels": {
             "type": "string",
             "default": "numbers"
-        }
-    },
-    "additionalProperties": false
-}
+        },
+        "default_statuses": {
+            "type": "object",
+            "properties": {
+                "FloorPlanTile": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string"
+                            },
+                            "color": {
+                                "type": "string"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "additionalProperties": false
+    }
```

Comment:
Is this change correct/intended or should "additionalProperties" be at one level less like it was before?